doc/nrf/libraries/nfc/lib_nfc_ndef.rst

```diff
@@ -0,0 +1,28 @@
+.. _lib_nfc_ndef:
+
+NFC Data Exchange Format (NDEF)
+###############################
+
+NFC communication uses NFC Data Exchange Format (NDEF) messages to exchange data.
+
+See the :ref:`ug_nfc` user guide for more information about :ref:`ug_nfc_ndef`.
+
+The |NCS| provides modules for generating messages and records to make it easy to generate such messages.
+If you want to implement your own polling device, you can use the supplied parser libraries.
+
+.. note::
+   You need additional hardware for reading NFC tags.
+
+Currently, the following messages and records are supported:
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   ../../include/nfc/ndef/*
+
+
+The NFC NDEF libraries are used in the following samples:
+
+* :ref:`record_text`
+* :ref:`writable_ndef_msg`
```

Comment:
I think also these can be added to the list (they all use NDEF libraries):
- nfc_tag_reader
- nfc_tnep_tag
- nfc_tnep_poller